descriptors/ModuleDescriptor-template.json

```diff
@@ -488,13 +488,13 @@
       ]
     },
     {
-      "id": "instance-bulk",
+      "id": "record-bulk",
       "version": "0.1",
       "handlers": [
         {
           "methods": ["GET"],
-          "pathPattern": "/instance-bulk/ids",
-          "permissionsRequired": ["inventory-storage.instance-bulk.ids.get"]
+          "pathPattern": "/record-bulk/ids",
```

Comment:
This is a breaking change, once you merge this UI will not be able to download any UUIDs, block this PR and let UI devs make necessary changes and only then merge this PR

Reply:
@KVupp yes, I described it in notes